holoviews/plotting/bokeh/renderer.py

```diff
@@ -93,3 +102,20 @@ def get_size(self_or_cls, plot):
         Returns a tuple of (width, height) in pixels.
         """
         return (plot.state.height, plot.state.height)
+
+
+    @classmethod
+    def validate(cls, options):
+        """
+        Validates a dictionary of options set on the backend.
+        """
+        try:
+            import bokeh
+            import bokeh.io
+        except:
+            raise ImportError("Could not import one of bokeh, pandas or scipy.")
+
+        if not cls._loaded:
+            bokeh.io.load_notebook()
```

Comment:
I am happy for this to be here as long as this is good behavior for injecting the bokeh javascript (i.e fewer problems with deleted cells, or repeating the same javascript over and over etc).